### PR TITLE
Add sourcemap and pretty-printing mochitests

### PR DESCRIPTION
**Pull request by jlongster**
This is a starting point for some more tests. There are still other edge cases we should test, but some things are undefined behavior until we know what UX we want. For example, things like what happens when you set a breakpoint on a generated source that has a sourcemap? Right now, it moves the breakpoint to the original source, but I'm not sure we want that behavior.

This covers the most serious uses though. Many of the other mochitests in the old debugger aren't applicable anymore either because we have a different UX or we don't have certain features yet (like blackboxing).

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ if (isFirefoxPanel()) {
     destroy: () => {
       unmountRoot();
       sourceMap.destroyWorker();
-      prettyPrint.destoryWorker();
+      prettyPrint.destroyWorker();
```
Comment (jasonLaster) [inline public/js/main.js]: what's the change here? I've been seeing `destroyWorker` does not exist.

Comment (clarkbw) [inline public/js/main.js]: destroy not destory 😀

Comment (jasonLaster) [inline public/js/main.js]: Ahhhh. I see :)

Comment (jlongster) [inline public/js/main.js]: Yep!

**Comment by jlongster** [comment]
Triggering it again on CI to make sure it passes again, then will merge.

**Comment by jasonLaster** [comment]
Great!
